### PR TITLE
Ng update

### PR DESCRIPTION
**Pull request by rostag**

### CODE REVIEW

File: src/app/shared/project/project.service.ts [hunk 1]
```diff
@@ -182,18 +181,15 @@ export class ProjectService {
...
-  bulkDeleteProjects(ids: string[]): Observable<IProject> {
+  bulkDeleteProjects(ids: string[]): Observable<Object> {
     const headers = new HttpHeaders().set('Content-Type', 'application/json');
     const body = JSON.stringify({ ids: ids });
 
-    return this.http.put(this.projectApiUrl + 'bulk-delete', body, { headers: headers })
-      .pipe(
-        catchError(this.handleError)
-      );
+    return this.http.put(this.projectApiUrl + 'bulk-delete', body, { headers: headers });
```
Comment (LimarenkoDenis) [inline src/app/shared/project/project.service.ts]: pls use es6 template syntax `${...}`

File: src/app/shared/task/task.service.ts [hunk 2]
```diff
@@ -108,17 +113,10 @@ export class TaskService {
    * @param ids {Array} Task IDs to update
    * @param data {Object} The data to be applied during update in {field: name} format
    */
-  bulkUpdateTasks(ids: Array<string>, data: any): Observable<ITask> {
+  bulkUpdateTasks(ids: Array<string>, data: any): Observable<Object> {
```
Comment (LimarenkoDenis) [inline src/app/shared/task/task.service.ts]: use specific type, not Object

Comment (rostag) [reply src/app/shared/task/task.service.ts]: Thanks, it had issues due to lib versions sync, now it's gone. Fixed.

File: src/app/shared/project/project.service.ts (hunk 1 above)
Comment (rostag) [reply src/app/shared/project/project.service.ts]: Good point!

**Pull request merged**